csharp/Google.Protobuf.Tools.targets

```diff
@@ -0,0 +1,16 @@
+<Project>
+    <PropertyGroup>
+        <protoc_linux64>$(MSBuildThisFileDirectory)..\tools\linux_x64\protoc</protoc_linux64>
+        <protoc_linux86>$(MSBuildThisFileDirectory)..\tools\linux_x86\protoc</protoc_linux86>
+        <protoc_macosx64>$(MSBuildThisFileDirectory)..\tools\macosx_x64\protoc</protoc_macosx64>
+        <protoc_macosx86>$(MSBuildThisFileDirectory)..\tools\macosx_x86\protoc</protoc_macosx86>
+        <protoc_windows64>$(MSBuildThisFileDirectory)..\tools\windows_x64\protoc.exe</protoc_windows64>
+        <protoc_windows86>$(MSBuildThisFileDirectory)..\tools\windows_x86\protoc.exe</protoc_windows86>
+        <protoc Condition="'$([MSBuild]::IsOsPlatform(Linux))' And '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)'=='X64'">$(protoc_linux64)</protoc>

```

Comment:
[MSBuild]::IsOsPlatform() requires MSBuild 17.3. So these checks are compatible only with the latest dotnet and VS 2017 with recent updates. No Mono, no VS2015. Is this ok?

Reply:
And [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture() is defined only in .NET Framework 4.7.1+.

This change is very breaking!